docs/usage/configuration-options.md

```diff
@@ -1239,6 +1239,11 @@ Example:
 
 The above would mean Renovate would not include files matching the above glob pattern in the commit, even if it thinks they should be updated.
 
+## expandCodeownersGroups
+
+If configured, Renovate will expand the configured code owner groups into a full list of group members and assign them individually.
```

Comment:
```suggestion
If configured, Renovate will expand any matching `CODEOWNERS` groups into a full list of group members and assign them individually instead of the group.
```